ovos_config/config.py

```diff
@@ -46,7 +47,7 @@ class Configuration(dict):
     remote = RemoteConf()
     # This includes both the user config and
     # /etc/xdg/mycroft/mycroft.conf
-    xdg_configs = [LocalConf(p) for p in get_xdg_config_locations()]
```

Comment:
These are 2 different configurations; this change appears to load `ovos/ovos.conf` files, rather than the configured base directory (default `mycroft/mycroft.conf`).